docs/caveats.md

```diff
@@ -280,6 +280,10 @@ We're not testing Fortinet implementation as part of the regular integration tes
 * Junos cannot have more than one loopback interface per routing instance. Using **loopback** links on Junos devices will result in configuration errors.
 * Junos configuration template configures BFD timers within routing protocol configuration, not on individual interfaces
 
+(caveats-linux)=
```

Comment:
All other Linux caveats are in 'labs/linux.md'. Please move this one there and insert a pointer in this file (all other documentation should also point to that file, and at least some of the platform tables do).

Reply:
done